geoalchemy2/elements.py

```diff
@@ -113,6 +113,35 @@ def desc(self):
         return binascii.hexlify(self.data)
 
 
+class EWKBElement(_SpatialElement, functions.Function):
+    """
+    Instances of this class wrap a EWKB value. Geometry values read
+    from the database are converted to instances of this type. You may
+    need to create instances of ``EWKBElement`` yourself.
```

Comment:
The `WKBElement` docs say that "In most cases you won't need to create WKBElement instances". Why do the `EWKBElement` docs say something different?